doc/cygnus-ngsi/installation_and_administration_guide/install_from_sources.md

```diff
@@ -134,6 +135,13 @@ In both cases, the installation is completed by copying the `cygnus-flume-ng` sc
 
 [Top](#top)
 
+###<a name="section4.4"></a>Known issues
+It may happen while compiling either `cygnus-common` either `cygnus-ngsi` the Maven JVM has not enough memory. This can be chaged as detailed at the [Maven official documentation](https://cwiki.apache.org/confluence/display/MAVEN/OutOfMemoryError):
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/955/commits/626cdcc1c89100bd68ad03149cd7cd4223add6e9